src/components/calendar/calendar.jsx

```diff
@@ -127,58 +242,58 @@ function Calendar(props) {
         </thead>
         <tbody>
           <tr>
-            {getCell(iterator, month, highlightedDate)}
-            {getCell(iterator, month, highlightedDate)}
-            {getCell(iterator, month, highlightedDate)}
-            {getCell(iterator, month, highlightedDate)}
-            {getCell(iterator, month, highlightedDate)}
-            {getCell(iterator, month, highlightedDate)}
-            {getCell(iterator, month, highlightedDate)}
+            {getCell(iterator, month, highlightedDate, refs)}
```

Comment:
This parameter list does not match the actual function signature!

Reply:
it was for a little bit there!! 😅 Apparently JS doesn't care soooooooooooooo ... 🤷 

/s